stable/openvpn/Chart.yaml

```diff
@@ -3,7 +3,7 @@ description: A Helm chart to install an openvpn server inside a kubernetes clust
   generation is also part of the deployment, and this chart will generate client keys
   as needed.
 name: openvpn
-version: 3.8.0
+version: 3.8.1
```

Comment:
This is changing functionality so should be a minor update rather than a patch which would make this 3.9.0.  see [semver](https://semver.org/#summary).

Reply:
thanks for the info, corrected